src/priceClient/adapters/acrossApi.ts

```diff
@@ -2,12 +2,17 @@ import { msToS, PriceFeedAdapter, TokenPrice } from "../priceClient";
 import { BaseHTTPAdapter } from "./baseAdapter";
 
 type AcrossPrice = { price: number };
+type AcrossApiArgs = {
+  name?: string;
+  host?: string;
+  timeout?: number;
+};
 const defaultTimeout = 5000; // mS
 
 export class PriceFeed extends BaseHTTPAdapter implements PriceFeedAdapter {
-  constructor(name: string, { host, timeout }: { host?: string; timeout?: number }) {
+  constructor(args?: AcrossApiArgs) {
     // Allow host to be overridden for test or alternative deployments.
-    super(name, host ?? "across.to", { timeout: timeout ?? defaultTimeout });
+    super(args?.name ?? "Across API", args?.host ?? "across.to", { timeout: args?.timeout ?? defaultTimeout });
```

Comment:
nb. This does read a _little_ bit complicated; was the most sensible thing I could figure out, but there might be better/cleaner ways of doing it.

Reply:
I think there's a simpler way with just a default arg unless I'm missing something!